js/autotrack/reactNavigation.js

```diff
@@ -29,7 +29,7 @@ export const withReactNavigationAutotrack = track => AppContainer => {
 
       track(EVENT_TYPE, {
         path: initialPageviewPath,
-        type: INITIAL_ROUTE_TYPE,
```

Comment:
Remind me - what's the motivation for this change?  Consistency with something else, or is `type` a special keyword somewhere?

Reply:
`type` represents the event type (param `t` in the pixel).  So if we pick a `type` field from `sprops` and throw it into the non-custom part of the event getting written to kafka, this field would overwrite the *real* `type`.  This used to not be an issue, since `type` was treated as a custom prop.